### PR TITLE
Update Wheelright background image

### PR DESCRIPTION
**Pull request by luke9kim8**
## Update Wheelright background image

Issue Number(s): #119 

What does this PR change and why?
* Update Wheelright title screen image with the one they gave us.

### Checklist

- [x]  Database schema docs have been updated or are not necessary
- [x]  Code follows design and style guidelines
- [x]  Code is commented with doc blocks
- [x]  Tests have been written and executed or are not necessary
- [x]  Latest code has been rebased from base branch (usually `develop`)
- [x]  Commits follow guidelines (concise, squashed, etc)
- [x]  Github issues have been linked in relevant commits
- [x]  Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes
* I'm always critical

### Related PRs
* N/A

### How to Test
* Check the title screen on mobile and desktop for Wheelright

### CODE REVIEW

File: src/pages/StartMobile/StartMobileWheelright.js [hunk 1]
```diff
@@ -4,12 +4,11 @@ import '../../static/css/Start.css';
...
 import '../../static/css/commons.css';
 import '../../static/css/StartMobile.css';
-import wheelrightPage1 from '../../static/images/wheelright-page1.jpg';
 
 const StartMobileWheelright = ({ setWheelright }) => {
   return (
-    <div className="start-body-mobile">
-      <img className="start-wheelright-background" src={wheelrightPage1} alt="frontpage" />
+    <div className="start-body-mobile wheelright-title-background">
+      {/* <img className="start-wheelright-background" src={wheelrightPage1} alt="frontpage" /> */}
```
Comment (kavinphan) [inline src/pages/StartMobile/StartMobileWheelright.js]: remove commented out code pls

File: src/static/css/Start.css [hunk 2]
```diff
@@ -292,7 +292,7 @@
 /* WheelRight below */
 
 .start-title-background {
-  background-image: url('../images/wheelright-page1.jpg');
+  background-image: url('../images/wheelright-background.JPG');
```
Comment (kavinphan) [inline src/static/css/Start.css]: why is the extension capitalized? 🤔

**Review by kavinphan — CHANGES_REQUESTED** [review]
why are the images red?
Screen Shot 2022-04-15 at 8 30 54 PM

File: src/pages/StartMobile/StartMobileWheelright.js (hunk 1 above)
Comment (luke9kim8) [reply src/pages/StartMobile/StartMobileWheelright.js]: yuh yuh

File: src/static/css/Start.css (hunk 2 above)
Comment (luke9kim8) [reply src/static/css/Start.css]: changed it

**Review by kavinphan — APPROVED** [review]

**Pull request merged**